src/js/pages/room/index.js

```diff
@@ -529,10 +529,37 @@ const Settings = ({open}) => {
   // Hooks
   //
   const [screen, setScreen] = useState(SETTINGS_SCREEN_MENU);
+  const [codeCopied, setCodeCopied] = useState(false);
+  const inviteCode = useSelector(BuoySelectors.connectedBuoy).get('token');
+
+  ////
+  // Action callbacks
+  //
+  const copyInviteCode = () => {
```

Comment:
I went this route because I thought it looked better not to have the large textarea in the settings page with the invite code in it. If there's a better way to do this though I'm all for it. :)

Reply:
Yeah, this is fine. You might want to try using the [HTML5 Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) but if this works it works 👍 .